README.md

```diff
@@ -1,2 +1,10 @@
 # InteractiveProgramming
-This is the base repo for the interactive programming project for Software Design, Spring 2016 at Olin College.
+In order to run Durak, you must clone the repo to you computer. After that, enter the repo and run:
+
```

Comment:
You are missing explanation of what you plan on doing in this project and what each individual's learning goals are.